agenta-backend/agenta_backend/routers/testset_router.py

```diff
@@ -146,7 +188,10 @@ async def update_testset(testset_id: str, csvdata: NewTestset):
 
 
 @router.get("/")
-async def get_testsets(app_name: Optional[str] = None):
+async def get_testsets(
+    app_name: Optional[str] = None,
+    stoken_session: SessionContainer = Depends(verify_session()),
+):
```

Comment:
I think we should add the output of the function in the typing here

Reply:
Sure!